airflow/ui/src/pages/Run/Details.tsx

```diff
@@ -118,7 +118,7 @@ export const Details = () => {
             </Table.Row>
             {dagRun.external_trigger ? (
               <Table.Row>
-                <Table.Cell>Externally Trigger Source</Table.Cell>
+                <Table.Cell>External Trigger Source</Table.Cell>
```

Comment:
@pierrejeambrun @bbovenzi We should nix this "external" word here, as some of the values are "timetable", or "asset" which aren't external.

Reply:
Do we always want to show `dagRun.triggered_by` then? instead of only on external triggers?

Reply:
While we decide on that we can at least remove the `External` word from here as this is misleading:
https://github.com/apache/airflow/pull/46559